ReactSkia/sdk/ThreadSafeCache.h

```diff
@@ -0,0 +1,49 @@
+/*
+* Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+#include <mutex>
+#include <better/map.h>
+#include "ReactSkia/utils/RnsLog.h"
+
+using namespace std;
+namespace facebook {
+namespace react {
+
+template <typename KeyT, typename ValueT>
+class RSkThreadSafeCache {
+ private:
```

Comment:
can we add comment for the variables, what each value represents .
It will be helpful for using the template in other modules/components